tests/testthat/test_all.R

```diff
@@ -0,0 +1,1245 @@
+tifFile <- downloadMouseBrainImage()
+
+annots <- system.file("extdata", "muBrain_LabWorksheet.txt", 
+                      package = "SpatialOmicsOverlay")
+
+annotsDF <- readLabWorksheet(lw = annots, slideName = "4")
+
+##################################  Utils  #####################################
+testthat::test_that("mouse brain tiff can be downloaded",{
```

Comment:
I'm guessing specs will be updated later as this is just a doc update.

Reply:
Yep I can update specs tomorrow. some of the specs will be tested in the longtests, fyi